app/models/annotation/Annotation.scala

```diff
@@ -273,7 +273,7 @@ object AnnotationDAO extends SQLDAO[Annotation, AnnotationsRow, Annotations] {
                _team = ${a._team.id},
                _user = ${a._user.id},
                skeletonTracingId = ${a.skeletonTracingId},
-               volumeTracingId = ${a.skeletonTracingId},
+               volumeTracingId = ${a.volumeTracingId},
```

Comment:
@fm3 Tracing a task was broken because of this. This fixed the bug for me. Please double check (tomorrow's fine), whether this is ok :)